src/components/ContactGroups/SelectBox/EditGroupName.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Input from 'cozy-ui/transpiled/react/Input'
+
+const EditGroupName = ({ groupName, setEditedGroupId }) => {
+  const stopPropagation = e => {
+    e.stopPropagation()
+  }
+
+  const exitEditMode = () => setEditedGroupId('')
+
+  const handleKeyDown = async e => {
+    // enter key
+    if (e.keyCode == 13) {
+      exitEditMode()
+      // TODO : Add function to save new group name
+    }
+    stopPropagation(e)
+  }
+
+  return (
+    <Input
+      id={'editGroupInput'}
+      type="text"
+      defaultValue={groupName}
+      size="tiny"
+      autoComplete="off"
+      autoFocus={true}
+      onKeyDown={handleKeyDown}
+      onClick={stopPropagation}
+      onFocus={stopPropagation}
+      onMouseDown={stopPropagation}
```

Comment:
Why does everything has stop propagation ? Is it because were are in an ActionMenu ?

Reply:
In a SelectBox in a modal. If the propagations are not stopped, there are side effects (backward in the browser when I delete a character for example).